### PR TITLE
fix target name

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
Fix target name for the Proxy object

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: plural/helm/console/templates/proxies.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ metadata:
 {{ include "console.labels" . | indent 4 }}
 spec:
   type: sh
-  target: statefulset/console
```
Comment (DavidSpek) [inline plural/helm/console/templates/proxies.yaml:27]: This should actually be `deployment/console`. The console used to be a statefulset but it was changed to a deployment. The `plural-console` statefulset is the postgres database.

**Review by DavidSpek — CHANGES_REQUESTED** [review]
The current proxy is indeed broken but it needs a different fix. Also, can you bump the helm chart version as well.

File: plural/helm/console/templates/proxies.yaml (hunk 1 above)
Comment (zreigz) [reply plural/helm/console/templates/proxies.yaml:27]: done

**Review by DavidSpek — APPROVED** [review]
LGTM

**Pull request merged**